Cargo.toml

```diff
@@ -212,8 +212,9 @@ tokio-postgres = { version = "0.5.5", features = ["runtime", "with-chrono-0_4"],
 toml = "0.5.8"
 typetag = "0.1.6"
 url = "2.2.0"
-uuid = { version = "0.8", features = ["serde", "v4"], optional = true }
+uuid = { version = "0.8", features = ["serde", "v4"] }
```

Comment:
I realise I'm undoing a recent PR to add `uuid` as optional. The crate size is ~37kb, so I don't think this adds much bloat. This prevents having to add feature guards in a few places. If this is contentious, I can add the guards.